code/modules/uplink/uplink_items.dm

```diff
@@ -449,7 +449,7 @@ GLOBAL_LIST_INIT(uplink_items, subtypesof(/datum/uplink_item))
 	desc = "Though capable of near sorcerous feats via use of hardlight holograms and nanomachines, they require an \
 			organic host as a home base and source of fuel. Holoparasites come in various types and share damage with their host."
 	item = /obj/item/storage/box/syndie_kit/guardian
-	cost = 18
+	cost = 20 //SKYRAT EDIT CHANGE - ORIGINAL: 18 - Stupidly powerful considering traitors get 35tc now.
```

Comment:
No. Change this to 10tc like holocarps are

Reply:
That's

no

These are insanely different from holocarp.

Reply:
Turns out I'm wrong.

Therefore, buff holocarp to match, 10tc for holoparas are stupidly powerful

Reply:
^ has not read the bottom post of all of this, or done any form of damage calculations and is going to be embarrassed in dev-talk